### ISSUE TITLE
Any visualization tool to visualize the training loss?

### ISSUE THREAD

**Issue by zshyang** [facebookresearch/meshrcnn#42]
As we could see during training there are many prints out. 
But without plot the loss curve it is hard to say what happened.
So do you have any plot tool for this purpose?

**Comment by gkioxari** [comment]
We print the total training loss and even break it down to task-specific losses for you. And while we don't provide any tools for it, plotting the losses given this info is trivial.

**Comment by zshyang** [comment]
I see. Thanks!
